src/UCTSearch.h

```diff
@@ -76,6 +76,7 @@ class UCTSearch {
 
     UCTSearch();
     void set_gamestate(const GameState& g);
+    GameState get_gamestate() const;
```

Comment:
You aren't using this as far as I can see? (And if you were, shouldn't it return a ref?)

Reply:
It's used by the gtest version. I will remove it from this pull and have it only in the gtest pull version.